js/spec/index-spec.js

```diff
@@ -891,6 +891,223 @@ describe('Node Sentinel File Watcher', function() {
       }
 
     });
+
+    it('can remove an excluded path', async function () {
+      const T1Path = path.join(workDir, 'T1');
+      await fse.mkdir(T1Path);
+      const T2Path = path.join(T1Path, 'T2');
+      await fse.mkdir(T2Path);
+      const T3Path = path.join(T2Path, 'T3');
+      await fse.mkdir(T3Path);
+
+      const newFile = 'newfile.file';
+      const newFilePath = path.join(T2Path, newFile);
+
+      let changeEvents = 0;
+      let createEvents = 0;
+      let deleteEvents = 0;
+
+      function findEvents(element) {
+        if (
+          element.action === nsfw.actions.MODIFIED && element.file === newFile
```

Comment:
I'm curious why we need to check for `newFile` here: what other files are we modifying while the test is running that's getting picked up?

Reply:
Because we receive events about that the directory that contains the file has changed, this behavior is different by SO.